python/chronos/src/bigdl/chronos/autots/autotsestimator.py

```diff
@@ -91,7 +91,7 @@ def __init__(self,
                tf.keras optimizer instance.
         :param past_seq_len: Int or or hp sampling function. The number of historical steps (i.e.
                lookback) used for forecasting. For hp sampling, see bigdl.orca.automl.hp for more
-               details. The values defaults to 2.
+               details. The values defaults to 'auto'.
```

Comment:
The values defaults to 'auto', which will automatically infer the cycle length of each time series and take the mode of them. The search space will be automatically set to hp.randint(0.5\*cycle_length, 2\*cycle_length)